public/views/ButtonView.js

```diff
@@ -0,0 +1,22 @@
+var Backbone = require('backbone');
+
+var eventHandler = require('../events/eventHandler');
+
+module.exports = Backbone.View.extend({
+  el: '#buttons',
+  render: function() {
+    $(this.el).html('<p class="soon-to-plant"><a href="/planting">Planting Soon List</a></p>');
+    // href="/planting"
+    return this;
+  },
+  events: {
+    'click .soon-to-plant': this.displaySoonToPlantPlants
```

Comment:
The selector for this event should be on the a tag, not the class on the parent. Additionally, the action should just the the method name in quotes, instead of this.methodName:

```
events: {
  'click a': 'displaySoonToPlantPlants'
}
```